flash/vision/classification/model.py

```diff
@@ -25,12 +25,25 @@
 _resnet_backbone = lambda model: nn.Sequential(*list(model.children())[:-2])  # noqa: E731
 _resnet_feats = lambda model: model.fc.in_features  # noqa: E731
 
+_densenet_backbone = lambda model: nn.Sequential(*model.features, nn.ReLU(inplace=True))
```

Comment:
I feel it would be better to bundle them up into one function, with multiple conditions.